src/test/java/net/tests/app/examples/MultipleWindows.java

```diff
@@ -0,0 +1,29 @@
+package net.tests.app.examples;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+
+import java.lang.reflect.Array;
+
+public class MultipleWindows {
+
+    public static void main (String args[]){
+        System.setProperty("webdriver.chrome.driver","src/drivers/chromedriver");
```

Comment:
If you want you can add some comments that help you remember what you're doing in the code, maybe the newest things or those that you were having doubts about.